templates/about.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "base.jinja2" %}
+{% block body %}
+  <section id="about-text">
+    <div class="container">
+      <article>
+        <h1>Learning Journal</h1>
+        <p>This project was created as part of the pre-work for the <a href="https://www.codefellows.org/development-accelerators/">Code Fellows Python Development Accelerator</a>.</p>
+        <p>Currently this stands as a mockup for style and layout purposes. <a href="http://sass-lang.com">Sass</a> was used to facilitate stylesheet creation, and effort was made towards a responsive design. Although no CSS frameworks are used directly, inspiration for the CSS organization was heavily inspired by <a href="http://getskeleton.com">Skeleton</a>.</p>
+        <p>More details will be added as this project is developed.</p>
+      </article>
+    </div>
+  </section>
+{% endblock %}
```

Comment:
This probably ought to be re-written too, now that the journal is live and functioning.